WooCommerce/Classes/POS/ViewModels/PointOfSaleDashboardViewModel.swift

```diff
@@ -47,11 +61,41 @@ final class PointOfSaleDashboardViewModel: ObservableObject {
     func showFilters() {
         showsFilterSheet = true
     }
+
+    func calculateAmounts() {
+        // TODO: this is just a starting point for this logic, to have something calculated on the fly
+        if let formattedCartTotalPrice = formattedCartTotalPrice,
+           let subtotalAmount = currencyFormatter.convertToDecimal(formattedCartTotalPrice)?.doubleValue {
+            let taxAmount = subtotalAmount * 0.1 // having fixed 10% tax for testing
+            let totalAmount = subtotalAmount + taxAmount
+            formattedOrderTotalTaxPrice = currencyFormatter.formatAmount(Decimal(taxAmount))
+            formattedOrderTotalPrice = currencyFormatter.formatAmount(Decimal(totalAmount))
+        }
+    }
```

Comment:
Added this so we have some kind of live updating of amounts before we call/calculate for proper subtotal and tax amounts

Reply:
Thanks for adding the commits to pull the latest trunk and add this dynamic placeholder value 👍 

Nit suggestions:
- This can be private, I updated it in https://github.com/woocommerce/woocommerce-ios/pull/12850/commits/e6330d57f35a2289b7af9a3c6caac9cfa1b0043c
- We can consider mapping the cart to the cart total price like in `PointOfSaleDashboardViewModel. observeProductsInCartForCartTotal` in this WIP branch](https://github.com/woocommerce/woocommerce-ios/compare/feat/12763-pos-order-totals?expand=1#diff-e51271ca4267e3faf4472077b6d50858da037a0c61aeb6e2dc3bfc0491f65df1R123-R138) to write it in a more reactive way vs. triggering from `didSet`